client/containers/lists/TechnologyListContainer.js

```diff
@@ -0,0 +1,27 @@
+import * as A from "../../actions";
+import * as COC from "../../components/componentConsts";
+import * as logger from "../../clientLogger";
+import {connect} from "react-redux";
+import TechnologyList from "../../components/lists/TechnologyList";
```

Comment:
Export TechnologyList from components/index.js and then import here from ../../components directly